src/suspendable/core.clj

```diff
@@ -25,8 +25,16 @@
 (defn- stop-missing-components [old-system new-system]
   (component/update-system old-system (missing-keys old-system new-system) component/stop))
 
+(defn- object? [x]
```

Comment:
Maybe instead of `object?` call this function something like `has-with-meta?` or `can-vary-meta?`.